packages/fxa-settings/src/pages/Index/index.tsx

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { IndexProps } from './interfaces';
+import AppLayout from '../../components/AppLayout';
+import CardHeader from '../../components/CardHeader';
+import InputText from '../../components/InputText';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import ThirdPartyAuth from '../../components/ThirdPartyAuth';
+import TermsPrivacyAgreement from '../../components/TermsPrivacyAgreement';
+import { isOAuthIntegration } from '../../models';
+import {
+  isClientMonitor,
+  isClientPocket,
+} from '../../models/integrations/client-matching';
+
+export const Index = ({
+  integration,
```

Comment:
What would you think about passing in `clientId`, `isSync`, `isOAuth` instead of the integration? It might remove the need for mocking in the index stories and unit tests.

Reply:
I see what you are saying since I didn't create a container component for this yet (I should have in hindsight!) which should be given the `integration`. I'm tempted to create it here, but it'd require story and test refactoring which we probably want to do anyway during implementation, so I think it's OK to do when we get to the functional ticket.